apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/[evaluationId]/page.tsx

```diff
@@ -37,15 +40,28 @@ export default async function ConnectedEvaluationPage({
     projectId: Number(params.projectId),
     uuid: params.commitUuid,
   })
-  const { rows, pagination } = await paginateQuery({
-    searchParams,
-    pageUrl: { base: pageUrl(params) },
-    dynamicQuery: computeEvaluationResultsWithMetadataQuery({
-      workspaceId: evaluation.workspaceId,
-      evaluation,
-      documentUuid: params.documentUuid,
-      draft: commit,
-    }).$dynamic(),
+  const rows = await computeEvaluationResultsWithMetadata({
+    workspaceId: evaluation.workspaceId,
+    evaluation,
+    documentUuid: params.documentUuid,
+    draft: commit,
+    page: searchParams.page as string | undefined,
+    pageSize: searchParams.pageSize as string | undefined,
+  })
+  const countResult = await computeEvaluationResultsWithMetadataCount({
```

Comment:
Maybe we could even do a separate api call. This is bellow in the page. No need to make the table wait for it. Also that will fix the count refresh issue when websockets have new results

Reply:
💯

Reply:
Yes, but that's in the current UI. Return a 0 here if you don't want to change anything and then do a count store where you do in frontend this computation

Reply:
yes will do something like this